### PR TITLE
Updated Durable Function Orchestrator Example to .NET 6

### PR DESCRIPTION
**Pull request by micheal-wheeler-1nebula**
###Overview:
- Updated to .NET 6
- Updated Nuget Packages to newest compatible versions
- Updated Service Bus integrations to use new Azure.Messaging.ServiceBus SDK
- Updated documentation for clarity

### CODE REVIEW

File: DurableFunctionOrchestratorExample/Constants/DurableFunctionOrchestratorConstants.cs [hunk 1]
```diff
@@ -6,12 +6,6 @@ public static class Constants
         public const string DurableOrchestrator = "DurableOrchestratorFunction";
         public const string OrchestrationTrigger = "OrchestrationTriggerFunction";
 
-        //Name of the configuration to fetch the connection string from
-        public const string ingressConnectionString = "ingressServiceBusConnectionString";
```
Comment (charloconnell) [inline DurableFunctionOrchestratorExample/Constants/DurableFunctionOrchestratorConstants.cs:10]: Were these just not used?

Comment (micheal-wheeler-1nebula) [reply DurableFunctionOrchestratorExample/Constants/DurableFunctionOrchestratorConstants.cs:10]: Yeah I think this was old stuff we missed when we did a config cleanup. I double checked and it's running fine without.

File: Orchestrator.Services/ServiceBusService.cs [hunk 2]
```diff
@@ -11,18 +11,18 @@ namespace Orchestrator.Services
 {
     public class ServiceBusService : IServiceBusService
     {
-        private readonly IQueueClient _queueClient;
+        private readonly ServiceBusClient _serviceBusClient;
```
Comment (charloconnell) [inline Orchestrator.Services/ServiceBusService.cs:14]: Does this need to be a field here? Since we currently just need the sender

Comment (micheal-wheeler-1nebula) [reply Orchestrator.Services/ServiceBusService.cs:14]: This is a hint to the devs to NOT dispose the object: 
image
https://learn.microsoft.com/en-us/dotnet/api/overview/azure/messaging.servicebus-readme?source=recommendations&view=azure-dotnet#client-lifetime
If it's disposed, the ServiceBusSender will also be disposed.

File: README.md [hunk 3]
```diff
@@ -3,7 +3,20 @@
...
+| Event     | Description                                               |
+| --        | --                                                        | 
+| Create    | User has signed up to your product via the marketplace    |
+| Update    | User has changed their plan or the number of seats        | 
+| Suspend   | User has failed to pay for the billing term               | 
+| Reinstate | User has resolved their billing issues                    | 
+| Delete    | User has cancelled their subscription                     | 
+
+An orchestrator can be any application that can receive and send messages via the AMQP 1.0 protocol. Connections can be achieved via [supported client libraries](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#client-libraries) that exist for multiple languages, [direct integrations](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#client-libraries) Microsoft and Azure-native services or the [Service Bus SDK](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-samples), which is also available for multiple platforms.
```
Comment (charloconnell) [inline README.md]: The "direct integration" link here goes to the client library anchor, and I think the sentence is missing a "with" after the the "direct integration"

File: Orchestrator.Services/ServiceBusService.cs (hunk 2 above)
Comment (micheal-wheeler-1nebula) [reply Orchestrator.Services/ServiceBusService.cs:14]: image
Better?

File: README.md (hunk 3 above)
Comment (micheal-wheeler-1nebula) [reply README.md]: image
Better?

**Review by charloconnell — APPROVED** [review]

**Review by charloconnell — APPROVED** [review]

**Pull request merged**